src/rules/avoid-dispatching-multiple-actions-sequentially.ts

```diff
@@ -26,13 +31,36 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
   },
   defaultOptions: [],
   create: (context) => {
+    const dispatchExpressionsWithSameParent = new Map<
+      TSESTree.Node,
+      TSESTree.CallExpression[]
+    >()
     return {
-      [multipleActionDispatch(readNgRxStoreName(context))](
+      [`${actionDispatch(readNgRxStoreName(context))}`](
         node: TSESTree.CallExpression,
       ) {
-        context.report({
-          node,
-          messageId,
+        if (
+          node.parent &&
+          isExpressionStatement(node.parent) &&
+          node.parent.parent
+        ) {
+          const nodes =
+            dispatchExpressionsWithSameParent.get(node.parent.parent) || []
+          dispatchExpressionsWithSameParent.set(node.parent.parent, [
+            ...nodes,
+            node,
+          ])
+        }
+      },
+      ['Program:exit']() {
+        dispatchExpressionsWithSameParent.forEach((dispatches) => {
+          if (dispatches.length > 1)
+            dispatches.forEach((node) =>
+              context.report({
+                node,
+                messageId,
```

Comment:
While reading this file I just noticed a small typo in the `messageId`:

https://github.com/timdeschryver/eslint-plugin-ngrx/blob/4d22e142cc4465f75376d5a2fc5e4a991b7e11c9/src/rules/avoid-dispatching-multiple-actions-sequentially.ts#L7

... it starts with a capitalized letter and all other `messageId` starts with a lowercase letter. Would it be good to change it to maintain the standard? :sweat_smile: